core/src/epicli/data/common/ansible/playbooks/roles/postgresql/defaults/main.yml

```diff
@@ -1,45 +1,48 @@
 ---
 pg:
-  service_name:
-    RedHat: postgresql-10
-    Debian: postgresql
-  instantiated_service_name:
-    RedHat: null
-    Debian: postgresql@10-main
   base_dir:
-    RedHat: /var/lib/pgsql
     Debian: /var/lib/postgresql
+    RedHat: /var/lib/pgsql
+  bin_dir:
+    Debian: /usr/lib/postgresql/10/bin
```

Comment:
Ad 1. IMO it does since it's easier to implement and test features when they are small. This approach also makes it easier to work together on dependent tasks.

Ad 2. For now I decided to keep the paths and service names not parametrized - for better readability (and copy + paste) but this can be changed when needed.